### ISSUE TITLE
Wrong "for" attribute generated in label tags 

### ISSUE THREAD

**Issue by EliuTimana** [brecons/bootstrap-tag-helper#33]
Hi, I have a model with this fields
```cs
    public class EppEquipoViewModel
    {
        public int? Id { get; set; }
        [DisplayName("Descripcion")]
        public string Descripcion { get; set; }
...
```
and this is my template
```cshtml

<form method="post" autocomplete="off" bc-validation="true">
    <form-group>
        <input asp-for="Descripcion" type="text" bc-help="Ingrese el nombre del equipo" bc-size="Small" bc-required="true"/>
    </form-group>
</form>
```
and this is what i get, i looks pretty good, but there is a problem with the generated html

![image](https://user-images.githubusercontent.com/11860802/47249723-bc498200-d3dc-11e8-9d6e-24a24a7400bd.png)

the generated `for` attribute in the label tag doesn't match with the input id, and this is a bad accessibility practice, this only happens on inputs that use the `asp-for` tag helper.
![image](https://user-images.githubusercontent.com/11860802/47249864-cff5e800-d3de-11e8-8911-2fc79e150a0a.png)

**Comment by EliuTimana** [comment]
A small workaround is to use the `asp-for` helper, it generates the right HTML
```cshtml

    <form-group>
        <label asp-for="Descripcion"></label>
        <input asp-for="Descripcion" type="text" bc-help="Ingrese el nombre del equipo" bc-size="Small" bc-required="true"/>
    </form-group>
```
but we lose the `*` when we use the `bc-required` attribute and we have to remove the `[DisplayName]` because it will generate a duplicate label
![image](https://user-images.githubusercontent.com/11860802/47249976-4515ed00-d3e0-11e8-9bea-7aa20751a33f.png)

**Comment by brecons** [comment]
Fixed with Pull Request #36
